scertecd/scertecd.go

```diff
@@ -752,7 +752,11 @@ func (cu *certUpdateCheck) finishACME(ctx context.Context, ci *acmeChallengeInfo
 			return nil, ctx.Err()
 		}
 		if oe, ok := err.(*acme.OrderError); ok {
-			cu.Logf("WaitOrder: OrderError status %q", oe.Status)
+			e := fmt.Sprintf("WaitOrder: OrderError status %q", oe.Status)
+			if order.Error != nil {
+				e += fmt.Sprintf(" error: %q", order.Error)
+			}
+			cu.Logf(e)
```

Comment:
oh, you don't want Logf now, because the first argument might contain a % from order.Error.

Or you want `cu.Logf("%s", e)`

But really I'd make order.Error != nil populate a string %s arg that's otherwise empty

Or just do

    cu.Logf("WaitOrder: OrderError status %q; err=%v", oe.Status, oe.Error)

Reply:
🤦 I had it as such in my first pass but rewrote as I was worried about nil Error pointer String method call. Thanks 🙂